docs/installation.rst

```diff
@@ -2,60 +2,34 @@
 Installation
 ============
 
-Installing with `conda <http://continuum.io/downloads>`_
+Installing with `conda <https://repo.anaconda.com/miniconda>`_
 --------------------------------------------------------
 
-From ``GMSO`` version ``0.3.0``, you can use `conda <http://continuum.io/downloads>`_ to install ``GMSO`` in your preferred environment. This will also install the dependencies of ``GMSO``.
+Starting from ``GMSO`` version ``0.3.0``, you can use `conda <https//repo.anaconda.com/miniconda>`_ to install ``GMSO`` in your preferred environment. This will also install the dependencies of ``GMSO``.
 ::
     (your-env) $ conda install -c conda-forge gmso
 
 
-
-Installing dependencies with `conda <http://continuum.io/downloads>`_
+Installing dependencies with `conda <https://repo.anaconda.com/miniconda>`_
 ---------------------------------------------------------------------
 
-Dependencies of ``GMSO`` are listed in the file ``requirements.txt``. They
-can be installed in one line:
-::
-
-    $ conda install -c omnia -c conda-forge --file requirements.txt
-
-Alternatively you can add all the required channels to your ``.condarc`` file
-and then install dependencies.
-::
-
-    $ conda config --add channels omnia
-    $ conda config --add channels conda-forge
-    $ conda install --file requirements.txt
-
-.. note::
-    These commands will likely change a configuration file on your computer and
-    may affect installation of other packages in other projects you are working
-    on. However, the channel priority recommended is fairly common
-    (in particular, ``conda-forge`` having the highest priority) and should
-    work well for most installations.
-
-Installing dependencies with `pip <https://pypi.org/project/pip/>`_
--------------------------------------------------------------------
-::
+Dependencies of ``GMSO`` are listed in the file ``environment.yml``. They
+can be installed in a ``conda`` environment named ``gmso`` with::
 
-    $ pip install -r requirements.txt
+    $ conda env create -f environment.yml
+    $ conda activate gmso
 
-.. note::
-    Compared to ``conda`` installation, this is less tested. Some upstream
-    dependencies may not be available on ``PyPI`` but can be installed via
-    source or ``conda``.
 
 Install an editable version from source
 ---------------------------------------
 
 Once all dependencies are installed, the ``GMSO`` itself can be installed.
-It is currently only available through its source code. It will be available
-through ``pip`` and ``conda`` in the future.
 ::
 
     $ git clone https://github.com/mosdef-hub/gmso.git
     $ cd gmso
+    $ conda env create -f environment-dev.yml
+    $ conda activate gmso-dev
     $ pip install -e .
```

Comment:
I feel like these could be combined... 🤔 Maybe something like 

Installing from source
----------------------
Dependencies of ``GMSO`` are listed in the files ``environment.yml`` (lightweight environment specification containing minimal dependencies) and ``environment-dev.yml`` (comprehensive environment specification including testing packages for developers). The ``gmso`` or ``gmso-dev`` ``conda`` environments can be created with::

Reply:
Then once the dependencies are installed install gmso::

Reply:
Let me know what you think of these changes!